src/World.js

```diff
@@ -36,6 +38,26 @@ class World {
                 })
             })
     }
+
+    segmentResults(eventSubgroupId) {
+        return this.request.protobuf(`/api/segment-results?world_id=0&segment_id=1&event_subgroup_id=${eventSubgroupId}&full=true&player_id=0`)
```

Comment:
I haven't tried leaving them out, but this is the way the Zwift client sends the request. I think segment id is probably required to be 1. This world ID doesn't appear to be the same as passed to the constructor. 0 may be a wild-card or something. I'm not sure about the player id.

Eventually I think I would want to add the ability to query segments as well as events, at which point the parameters may be needed.